plugin/illuminate.vim

```diff
@@ -20,6 +20,7 @@ lua << EOF
                 is_supported = require('nvim-treesitter.query').has_locals,
             }
         })
+        vim.g.illuminate_ts_module_defined = true
     end
```

Comment:
suggestion(perf): don't try to `require('nvim-treesitter')` at all if `vim.fn.has('nvim-0.9') == 1` (avoiding that call might improve startup time).